packages/formation/sass/base/_b-mixins.scss

```diff
@@ -74,7 +74,7 @@
     padding: 0;
     border: 0;
     margin: -1px;
-    content: "this will open a new website";
+    content: 'this will open a new website';
```

Comment:
Did this file need to change?

Reply:
No, got caught on my auto-linter. Reverted back to main branch, thanks!